crates/holochain/src/sweettest/sweet_conductor.rs

```diff
@@ -56,7 +56,7 @@ pub struct SweetConductor {
     /// since they are not persisted to the wasm database and therefore
     /// are not automatically loaded into the [`crate::conductor::ribosome_store::RibosomeStore`]
     /// on conductor restart otherwise.
-    dna_files: Vec<DnaFile>,
+    dna_files: Vec<(CellId, DnaFile)>,
```

Comment:
Probably better as a hash set, but not required on this PR.

Reply:
I switched to a HashMap now: https://github.com/holochain/holochain/pull/5189/commits/7519e5fed4e58df0281280b300e031c0fe8df159